dz_s8/zadacha_1.py

```diff
@@ -0,0 +1,27 @@
+# Напишите функцию, которая преобразует pickle файл
+# хранящий список словарей в табличный csv файл.
+# Для тестированию возьмите pickle версию файла из задачи
+# 4 этого семинара.
+# Функция должна извлекать ключи словаря для заголовков
+# столбца из переданного файла.
+
+import pickle
+import csv
+
+__all__ = ['from_pickle_to_csv', ]
+
+
+def from_pickle_to_csv(file_pickle, file_csv):
+    with(
+        open(file_pickle, 'rb') as fp,
+        open(file_csv, 'w', newline='', encoding='utf-8') as fc
+    ):
+        list_of_dict = pickle.load(fp)
+        all_keys = set([key for dict in list_of_dict for key in dict.keys()])
```

Comment:
Если у вас список словарей, то почему не сделать так:
```suggestion
        all_keys = list_of_dict[0].keys()
```